src/components/MangaCard.tsx

```diff
@@ -17,9 +17,8 @@ import SpinnerImage from 'components/util/SpinnerImage';
 
 const useStyles = makeStyles((theme) => ({
     root: {
-        height: '100%',
-        width: '100%',
         display: 'flex',
+        aspectRatio: '9/14',
```

Comment:
how about the respected `225/350`? I know it's the same ratio but it's the golden number when making manga...
also add a comment for future generations reading the code.